src/components/Link.tsx

```diff
@@ -0,0 +1,18 @@
+import { ComponentProps } from "react";
+import { Link as RouterLink } from "react-router-dom";
+import styles from "@kabisa/ui-components/src/atoms/Link/index.module.css";
+
+export type Props = ComponentProps<typeof RouterLink> & {
+  theme: "light" | "dark";
+};
+
+export const Link: React.FC<Props> = ({ theme, children, ...props }: Props) => {
+  return (
+    <RouterLink
+      {...props}
+      className={`${props.className} ${styles.link} ${styles[theme]}`}
+    >
+      {children}
+    </RouterLink>
+  );
+};
```

Comment:
Created new link component to use the link from react-router, combined with the styling of the kabisa link. In the previous solution, the links were nested, which was an illegal HTML construct.